.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,76 @@
+name: unit-tests
```

Comment:
`integration-tests`  would be a more accurate name (please also rename the file).

Reply:
Updated as integration-test